src/main/kotlin/org/partiql/scribe/sql/PartiQLValueTextWriter.kt

```diff
@@ -0,0 +1,375 @@
+package org.partiql.scribe.sql
+
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at:
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+import org.partiql.value.BagValue
+import org.partiql.value.BoolValue
+import org.partiql.value.CharValue
+import org.partiql.value.CollectionValue
+import org.partiql.value.DateValue
+import org.partiql.value.DecimalValue
+import org.partiql.value.Float32Value
+import org.partiql.value.Float64Value
+import org.partiql.value.Int16Value
+import org.partiql.value.Int32Value
+import org.partiql.value.Int64Value
+import org.partiql.value.Int8Value
+import org.partiql.value.IntValue
+import org.partiql.value.ListValue
+import org.partiql.value.MissingValue
+import org.partiql.value.NullValue
+import org.partiql.value.PartiQLValue
+import org.partiql.value.PartiQLValueExperimental
+import org.partiql.value.SexpValue
+import org.partiql.value.StringValue
+import org.partiql.value.StructValue
+import org.partiql.value.SymbolValue
+import org.partiql.value.TimeValue
+import org.partiql.value.TimestampValue
+import org.partiql.value.datetime.Date
+import org.partiql.value.datetime.Time
+import org.partiql.value.datetime.TimeZone
+import org.partiql.value.datetime.Timestamp
+import org.partiql.value.io.PartiQLValueWriter
+import org.partiql.value.util.PartiQLValueBaseVisitor
+import java.io.OutputStream
+import java.io.PrintStream
+import java.math.BigDecimal
+import kotlin.math.abs
+
+/**
+ * [PartiQLValueWriter] which outputs PartiQL text.
+ *
+ * TODO: Copied from partiql-lang-kotlin's [org.partiql.value.io.PartiQLValueTextWriter]. Delete after
+ *  https://github.com/partiql/partiql-lang-kotlin/issues/1491 is resolved and released as a new PartiQL Kotlin version.
```

Comment:
File is identical to the `PartiQLValueWriter` from https://github.com/partiql/partiql-lang-kotlin/pull/1492. Copied here until we release the updated `PartiQLValueWriter` in a new release.